wmcore_base/ContainerScripts/pyfutureTest.sh

```diff
@@ -15,19 +16,24 @@ git fetch origin pull/${ghprbPullId}/merge:PR_MERGE
 export COMMIT=`git rev-parse "PR_MERGE^{commit}"`
 git checkout -f ${COMMIT}
 
+echo "$(TZ=GMT date): running futurize first stage"
 futurize -1 src/ test/ > test.patch
```

Comment:
You can probably just do the same trick here. No need to check everything every time.